README.md

```diff
@@ -120,3 +120,97 @@ Developed with Typescript, Express & TypeORM.
     "message": "User deleted!"
   }
   ```
+
+---
+
+### Get all projects
+- Endpoint: GET `http://localhost:3000/project`
+- Headers:
+  - `jwt-token` => we should use as the value the token that we received from login.
```

Comment:
It could be better if we use Authorization key with Bearer prefix instead of 'jwt-token'.